persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConverationDAOImpl.kt

```diff
@@ -63,12 +67,16 @@ class ConversationDAOImpl(private val conversationQueries: ConverationsQueries,
     }
 
     override suspend fun insertMember(member: Member, conversationID: QualifiedID) {
-        memberQueries.insertMember(member.user, conversationID)
+        memberQueries.transaction {
+            userQueries.insertUser(member.user, null, null)
```

Comment:
So if the user already exists the name and handle will get nulled out since the insert query will overwrite any existing if it already exists. So maybe we shouldn't do an `upsert` in this case.

Reply:
Oversight on my side. I just thought order of sync would be enough, but this is too loose.

I added a new query to `userQueries` to insert or ignore.